mdx-models/src/main/java/com/mx/path/model/mdx/model/account/CoreFields.java

```diff
@@ -0,0 +1,63 @@
+package com.mx.path.model.mdx.model.account;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import com.mx.path.model.mdx.model.MdxBase;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+public class CoreFields extends MdxBase<CoreFields> {
+  private Boolean hasApr;
+  private Boolean hasApy;
+  private Boolean hasAvailableBalance;
+  private Boolean hasAvailableCredit;
+  private Boolean hasBalance;
+  private Boolean hasCashAdvanceApr;
+  private Boolean hasCashSurrenderValue;
+  private Boolean hasCreditLimit;
+  private Boolean hasCurrencyCode;
+  private Boolean hasDailyDepositLimitCurrent;
+  private Boolean hasDailyDepositLimitTotal;
+  private Boolean hasDayPaymentIsDue;
+  private Boolean hasDeathBenefit;
+  private Boolean hasMonthlyTransferLimit;
+  private Boolean hasHoldTotal;
+  private Boolean hasInsuredName;
+  private Boolean hasInterestPaidPreviousYear;
+  private Boolean hasInterestPaidYtd;
+  private Boolean hasInterestRate;
+  private Boolean hasIsClosed;
+  private Boolean hasIsHidden;
+  private Boolean hasLastPayment;
+  private Boolean hasLastPaymentDate;
+  private Boolean hasLoanAmount;
+  private Boolean hasMaturesDate;
```

Comment:
What is this field, just out of curiousity?

Reply:
https://developer.mx.com/drafts/mdx/accounts/#accounts-account-details-additional - these fields will represent our accounts fields but represents a single account. That way, mobile app can enable or disable certain fields based on the logic Path sets.

Reply:
Right, I just wasn't sure what a "mature" is in this context. `This account will show matures_on / matures_at date fields.`

Reply:
Oh okay, I think it's either of those based on whatever is provided by the client. Can they have both? 🤔